src/foundation.cs

```diff
@@ -7179,6 +7179,12 @@ interface NSString2 : NSSecureCoding, NSMutableCopying
 		[Static]
 		[Export ("localizedUserNotificationStringForKey:arguments:")]
 		string GetLocalizedUserNotificationString (string key, [Params] [NullAllowed] NSObject [] arguments);
+
+		[Export ("getParagraphStart:end:contentsEnd:forRange:")]
+		void GetParagraphPositions (out nuint paragraphStartPosition, out nuint paragraphEndPosition, out nuint contentsEndPosition, NSRange range);
+
+		[Export ("paragraphRangeForRange:")]
+		NSRange ParagraphRangeForRange (NSRange range);
```

Comment:
should be `NSRange GetParagraphRange (NSRange range);`, i.e.
* add a verb / action, prefix `Get`;
* remove suffix `ForRange` as it duplicate the parameter